kerl

```diff
@@ -642,6 +642,10 @@ maybe_patch_all() {
     if [ "$1" -ge 17 ] && [ "$1" -le 19 ]; then
         apply_zlib_patch >> "$LOGFILE"
     fi
+
+    if [ "$1" -ge 22 ] && [ "$1" -le 23 ]; then
```

Comment:
As far as I can tell, this patch **only** applies to mac OS - can you put this in `maybe_patch_darwin()` function which is around line 651?